RELEASE-NOTES.md

```diff
@@ -13,6 +13,8 @@ The Uberenv project release numbers follow [Semantic Versioning](http://semver.o
 - Allow projects to force specifying `--prefix` on command line via new project option:
   `force_commandline_prefix`.  This is useful when the default `uberenv_libs` can not exist
   inside of your project's source repository.
+- Adds support for Windows builds using [Vcpkg].
+- Adds the `--triplet` command line argument for setting the Vcpkg build configuration.
```

Comment:
such an odd naming.  i know it's right, it is just weird.

Reply:
especially since `x86-Windows` is the default "triplet"?
(I think the third one is the linkage -- static vs dynamic)